### ISSUE TITLE
Implement Array.sort() to stretch people who finish early

### ISSUE THREAD

**Issue by stevehopkinson** [ZooeyMiller/mc-array-methods-reduce#2]
We've had a chat about this elsewhere, but it would be fairly simple to add an additional challenge to implement Array.sort() for people who finish early. I imagine some students who're more familiar with array methods might fly through the current examples – sort is a meatier challenge for them to get their teeth into (and probably requires them to implement other methods first). 

On the other hand, you might think this is too much of a stretch/too intimidating/non-suitable – so feel free to ignore!

**Comment by shiryz** [comment]
I agree some people would finish the challenge really quickly, it can be added as a bonus method.

**Comment by ZooeyMiller** [comment]
I think this is a fair point. The worry I have for this challenge is that students who are already familiar with reduce/these array methods will fly through, so adding sort would be a good way to combat this.
